german/Leichen/Ausschau/Ausschau.md

```diff
@@ -0,0 +1 @@
+Da! Dort dr�ben in der Ferne! Leicht �ber der Wassermarke zeichnet sich etwas ab. Jedes Mal, wenn die wiegenden Wogen es zu lassen sp�ht ihr gen Strand. Vielleicht ein St�ck Treibholz, eine verwahrloste Boje? Ihr k�nnt euren Kopf kaum �ber Wasser halten und schluckt jede Menge Salzwasser, aber ihr seit euch sicher: wenn ihr es nur bis dort hin schafft, �berlebt ihr vielleicht.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.